dev-guide.md

```diff
@@ -86,12 +103,25 @@ Then we will get this JSON output from the AST renderer:
 }
 ```
 
-When passing this tree to a renderer, it is recursively traversed
+### Line numbers
+
+mistletoe records the starting line of all block tokens that it encounters during
+parsing and stores it the `line_number` attribute.
```

Comment:
```diff
-parsing and stores it the `line_number` attribute.
+parsing and stores it as the `line_number` attribute of each token.
```

Reply:
Fixed, along with the merge conflicts :)

Reply:
Perfect, thank you! :)